exporters/postgres/docker-compose.postgres-exporter.yml

```diff
@@ -9,6 +9,7 @@ services:
     image: prometheuscommunity/postgres-exporter:${POSTGRES_EXPORTER_VERSION:-latest}
     command:
       - '--config.file=/etc/postgres-exporter/postgres_exporter.yml'
+      - '--no-collector.replication_slot'
```

Comment:
Minor, but can you include a comment here (maybe even just a link to the issue) so that it is clear why we did this?

Reply:
Yup, good call!